lib/u3d/commands_generator.rb

```diff
@@ -127,6 +128,7 @@ def run
         c.summary = 'List download-ready versions of Unity'
         c.action do |_args, options|
           options.default packages: false
+          options.default no_central: false
```

Comment:
`central: true` given the above.

Also, do we need this in several commands (e.g. install)? Maybe make it global.